.github/workflows/im-services.yaml

```
@@ -0,0 +1,52 @@
+    paths:
+      - 'backend/municipal-services/im-services/**'
+  workflow_dispatch:
+  workflow_call:
+
+jobs:
+  docker_image-build:
+    if: (github.event_name == 'push' && (startsWith(github.ref, 'refs/heads/develop')))
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)